elasticsearch/examples/upgrade/Makefile

```diff
@@ -0,0 +1,16 @@
+default: test
+
+include ../../../helpers/examples.mk
+
+CHART := elasticsearch
+RELEASE := helm-es-upgrade
+FROM := 7.4.0	# versions before 7.4.O aren't compatible with Kubernetes >= 1.16.0
+TO := 7.10.0	# upgrade from 7.x to 8.0.0-SNAPSHOT currently doesn't work
```

Comment:
Are there any issues with the version numbers in these files getting updated by the version bumping automation?

Reply:
Good question!
- On master branch we don't do any bumping so we won't have any issues for sure.
- On release branches, we shouldn't need have issue because [community-release-tools](https://github.com/elastic/infra/blob/master/cd/release/community-release-tools) get old version from `Charts.yaml` and only bump this version.
  In addition we shouldn't need to set `TO` on release branches because release branches are using SNAPSHOT images from the same major stack version as the `FROM` version unlike master branch which is using 8.0.0 SNAPSHOT images.